include/zephyr/net/wifi_mgmt.h

```diff
@@ -51,8 +51,10 @@ extern "C" {
 
 /** Wi-Fi management commands */
 enum net_request_wifi_cmd {
+	/** Get Wi-Fi driver and Firmware version */
+	NET_REQUEST_WIFI_CMD_VERSION = 1,
 	/** Scan for Wi-Fi networks */
-	NET_REQUEST_WIFI_CMD_SCAN = 1,
```

Comment:
Do we really need to modify the existing enumeration? Can't we just add it at the end? I know version seems basic so, but it's ok to be at the last.

Reply:
okay, I thought it is basic first command for any Wi-Fi related activity.

Reply:
Updated